cypress/cypress/component/Datepicker/DatePickerInput.cy.tsx

```diff
@@ -45,14 +46,9 @@ it("setting intial value as iso string works", () => {
     [name]: yup.date().required(),
   });
 
-  const x = (values: unknown) => {
```

Comment:
@neotob 

I'm voting to disable `unicorn/consistent-function-scoping`

@manni497 your opinion?

Reply:
removed and reverted.